src/intro/hello-world.md

```diff
@@ -150,42 +295,44 @@ struct Player {
 
 Let's break this down.
 
-1. The gdext prelude contains the most common symbols. Less frequent classes are located in the `engine` module.
+1. We define two fields `speed` and `angular_speed` for the logic. These are regular Rust fields, no magic involved. More about their use later.
```

Comment:
The previous order was better, since it followed the order of different symbols appearing in the code.

I appreciate your improvements though (links + formulation), so would just need swapping some numbers 🙂